examples/fastapi/requirements.txt

```diff
@@ -1,3 +1,15 @@
-fastapi
 moesifasgi
 uvicorn
+passlib==1.7.4
+moesifapi==1.3.3
+moesifpythonrequest==0.2.0
+fastapi==0.78.0
+jose==1.0.0
+python-jose==3.3.0
+rsa==4.8
+ecdsa==0.17.0
+pyasn1==0.4.8
+cryptography==37.0.2
+cffi==1.15.0
+six==1.16.0
+pycparser==2.21
```

Comment:
I am not sure if all of above are being used in the code but that's fine for example codes.